test/e2e/specs/editor/various/validate-multiple-use.spec.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Validate multiple use', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should display correct amount of warning message', async ( {
+		editor,
+		page,
+	} ) => {
+		const OPTIONS_SELECTOR =
+			'//div[contains(@class, "block-editor-block-settings-menu")]//button[contains(@aria-label, "Options")]';
```

Comment:
Let's rewrite this (and others) to prefer `getByRole()` [from Playwright](https://playwright.dev/docs/intro)

Reply:
Hi @getdave 

The test is rewritten by playwright. 

`await expect( page.locator( '.block-editor-warning' ) ).toBeVisible();` is failing though the screenshot generated by playwright displaying the element. 

 

![test-failed-1](https://github.com/WordPress/gutenberg/assets/33577124/cb116bf1-b7d2-4fba-8a23-b818ddeb9a1f)

![image](https://github.com/WordPress/gutenberg/assets/33577124/669efdad-92b9-4553-ba8e-36c402652980)

**Any suggestions for this, Thanks in Advance**

cc: @Mamaduka

Reply:
My bad, needed to consider `iframe` in the selector. @getdave tests are rewritten by playwright and you were absolutely right, most of the await needed to be removed. Thanks for the suggestion 🙇

Would you be able to review again. Thanks